### PR TITLE
Search hotel cities around position

### LINKED ISSUE #12: Search hotelCities by geolocation
Add this information to the `Algolia` api. Naming must be as below

```
"_geoloc": {
      "lat": float,
      "lng": float
    }
```

Then add `latitude` and `longitude` query paramater to hotelCities query.

### PR DESCRIPTION
**Pull request by michalsanger**
close #12

### CODE REVIEW

File: src/hotel/dataloaders/HotelCities.js [hunk 1]
```diff
@@ -1,34 +1,62 @@
...
+            if (input.type === 'latLng') {
+              return citiesIndex.search({
+                aroundLatLng: `${input.lat}, ${input.lng}`,
+              });
+            }
+          }),
+        );
+
+        return responses.map(response => {
+          const cities = _.get(response, 'hits', []);
```
Comment (jaroslav-kubicek) [inline src/hotel/dataloaders/HotelCities.js]: no more `_.get`? 🙏 👼

Comment (michalsanger) [reply src/hotel/dataloaders/HotelCities.js]: removed here, created issue for whole repo #28

File: src/location/types/inputs/AreaInput.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { GraphQLInputObjectType, GraphQLNonNull } from 'graphql';
-import GraphQLCoordinates from './CoordinatesInput';
+import GraphQLCoordinates from '../../../common/types/inputs/CoordinatesInput';
```
Comment (mrtnzlml) [inline src/location/types/inputs/AreaInput.js]: Maybe it's time to start using Yarn workspaces or something like that... :)

Comment (michalsanger) [reply src/location/types/inputs/AreaInput.js]: Not in this PR OK?

Comment (mrtnzlml) [reply src/location/types/inputs/AreaInput.js]: Of course. It's quite a lot of work.

**Pull request merged**